e2e-tests/playwright/lib/src/server/plugin.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Client4} from '@mattermost/client';
+import {PluginManifest} from '@mattermost/types/plugins';
+
+export async function installAndEnablePlugin(
+    client: Client4,
+    pluginUrl: string,
+    pluginId: string,
+    force = true,
+): Promise<void> {
+    // Install from URL
+    await client.installPluginFromUrl(pluginUrl, force);
+
+    // Wait for installation
+    await new Promise((resolve) => setTimeout(resolve, 5000));
+
+    // Enable plugin
+    await client.enablePlugin(pluginId);
+
+    // Wait for activation
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
This is a bit arbitrary: should we try without and see how much tweaking is actually required in practice?